src/firmware/boards/robot_stm32h7/io/INA226_power_monitor_driver.h

```diff
@@ -21,8 +21,13 @@
  * Adapted from: https://github.com/macgeorge/STM32-example-codes
  */
 
+#pragma GCC diagnostic push
+#pragma GCC diagnostic ignored "-Wsign-conversion"
+#pragma GCC diagnostic ignored "-Wconversion"
```

Comment:
suggestion, perhaps for another pr: make a MACRO that disables all warnings, e.g. `-Wall,-Wextra,-Wconversion,-Wno-unused-parameter`

Reply:
I'll make a ticket for this, I can't seem to figure out a clean way to do this, looks like nested preprocessor directives aren't easy (maybe I'm just missing something)

Reply:
yeah the most obvious way is to have PREPROCESSOR_IGNORE_WARNINGS_START and PREPROCESSOR_IGNORE_WARNINGS_END macros, which isn't particularly elegant